react/features/base/tracks/functions.js

```diff
@@ -320,7 +320,7 @@ export function getTrackByMediaTypeAndParticipant(
         mediaType,
         participantId) {
     return tracks.find(
-        t => t.participantId === participantId && t.mediaType === mediaType
+        t => Boolean(t.jitsiTrack) && t.participantId === participantId && t.mediaType === mediaType
```

Comment:
without this change the connection indicator will crash, patching only the connection indicator leads to other issues...

Reply:
https://github.com/jitsi/jitsi-meet/blob/1041cd80553943ecea263549266b63757bc1d145/react/features/connection-indicator/components/web/ConnectionIndicator.js#L463